reflected_models.py

```diff
@@ -970,25 +969,26 @@ class Meta:
         managed = False
         db_table = 'adt_users'
 
-
+#AJ 09/07/2016...moved foreign key to the top
```

Comment:
For future commits, try to always put a space between your # and the text of the comment. That's the typical convention in Python. i.e. instead of ```#AJ``` do ```# AJ```